### ISSUE TITLE
Need way to protect JSON config file from being edited by non-admins

### ISSUE THREAD

**Issue by rchohrach-flowcal** [microsoft/azure-devops-extension-cascading-picklist#73]
We have discovered that any user in Azure DevOps can make changes to the JSON config file and save them.  We would like to limit this capability to only those users in the admin group.  We would like to prevent unauthorized/unapproved changes to the cascading picklists.  Is there currently any way to do this?

**Comment by ssteiner1** [comment]
Would like this functionality as well.

**Comment by StacyA1013** [comment]
I agree we do integration off our picklist cascading fields and being able to lock it to a specific Change management team would be great (ie not all Project Admins should have it)
